### ISSUE TITLE
Windows Downloadable Incomplete

### ISSUE THREAD

**Issue by TMIndust** [swig/swig#369]
The Windows prebuilt downloadable on swig.org is incomplete.
Took me a while to find out because the documentation clearly states
"Download the swigwin zip package from the SWIG website and unzip into a directory. This is all that needs downloading for the Windows platform."

Well the version is useless without a Lib-folder, you are only getting "can't find swig.swg" errors until you start comparing the stuff you have downloaded with the stuff on github and realize there are a lot of folders and files missing.
Basically every subfolder besides "vms" and "Win" are missing.

Well and now after downloading it again manually it tells me that it can't find swigwarn.swg (not swigwarnings.swg) which is also missing for some reason on Github. Great.
(Actually I just wanted to implement a library in my project and this is now the 5th program I have to learn first and still am at after several days, live is not easy.)

**Comment by ojwb** [comment]
http://www.swig.org/download.html links to http://prdownloads.sourceforge.net/swig/swigwin-3.0.5.zip - if I download that and check the contents, it doesn't match what you say above:

```
$ unzip -l swig-3.0.5.zip
Archive:  swigwin-3.0.5.zip
Unzip into a directory of your choice. Please read the README file as well as Doc\Manual\Windows.html for installation instructions.
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2015-02-01 00:00   swigwin-3.0.5/
        0  2015-01-31 23:58   swigwin-3.0.5/Source/
        0  2015-01-31 23:58   swigwin-3.0.5/Source/Include/
    12592  2015-01-31 17:51   swigwin-3.0.5/Source/Include/swigwarn.h
     2430  2015-01-31 23:58   swigwin-3.0.5/Source/Include/swigconfig.h.in
        0  2015-01-31 23:58   swigwin-3.0.5/Source/CParse/
     8468  2015-01-31 23:58   swigwin-3.0.5/Source/CParse/parser.h
    25447  2015-01-31 17:51   swigwin-3.0.5/Source/CParse/cscanner.c
     2852  2015-01-31 17:51   swigwin-3.0.5/Source/CParse/cparse.h
   427467  2015-01-31 23:58   swigwin-3.0.5/Source/CParse/parser.c
    27401  2015-01-31 17:51   swigwin-3.0.5/Source/CParse/templ.c
```

and so on, including `Lib`:

```
        0  2015-01-31 23:58   swigwin-3.0.5/Lib/
     1236  2015-01-31 17:51   swigwin-3.0.5/Lib/runtime.swg
      309  2015-01-31 17:51   swigwin-3.0.5/Lib/wchar.i
     3176  2015-01-31 17:51   swigwin-3.0.5/Lib/swiglabels.swg
        0  2015-01-31 17:51   swigwin-3.0.5/Lib/xml/
      179  2015-01-31 17:51   swigwin-3.0.5/Lib/xml/typemaps.i
       21  2015-01-31 17:51   swigwin-3.0.5/Lib/xml/xml.swg
        0  2015-01-31 17:51   swigwin-3.0.5/Lib/octave/
      679  2015-01-31 17:51   swigwin-3.0.5/Lib/octave/std_deque.i
    10673  2015-01-31 17:51   swigwin-3.0.5/Lib/octave/octruntime.swg
     3031  2015-01-31 17:51   swigwin-3.0.5/Lib/octave/director.swg
     3132  2015-01-31 17:51   swigwin-3.0.5/Lib/octave/std_pair.i
     4646  2015-01-31 17:51   swigwin-3.0.5/Lib/octave/std_map.i
```

and `swigwarn.swg` (which isn't in the git repo as it's a generated file):

```
    14012  2015-02-01 00:00   swigwin-3.0.5/Lib/swigwarn.swg
```

and so on...

**Comment by TMIndust** [comment]
Yes, they show up for me now also, after downloading it again.
Now I am very confused. This is random, very random.

I have still the .zip file I downloaded before and it definitly doesn't contain all this stuff which now appears all of a sudden in the new zip file.

Well... okay...

**Comment by ojwb** [comment]
The SF download system has a number of mirrors, and you'll get redirected to one automatically (probably one located "near" you, at least in some sense).  So it is possible one has a bad copy of the file, and you got a different mirror on the next try.

Or perhaps your first download was incomplete.

What size is the bad zip?

If your unzipper can verify a zip, does it verify OK?

@wsfulton Any ideas?
